static/ansi-to-html.ts

```diff
@@ -127,10 +127,27 @@ function generateOutput(stack: string[], token: string, data: string | number, o
     } else if (token === 'xterm256') {
         // Note: Param 'data' must be a string at this point
         return handleXterm256(stack, data as string, options);
+    } else if (token === 'rgb') {
+        // Note: Param 'data' must be a string at this point
+        return handleRgb(stack, data as string, options);
     }
     return '';
 }
 
+function handleRgb(stack: string[], data: string, options: AnsiToHtmlOptions) {
+    data = data.substring(2).slice(0, -1);
+    const operation = +data.substr(0, 2);
```

Comment:
I was going to request to change this to `substring` as `substr` has been deperecated for a while (In JS at least, no idea how TS handles this conversion if at all), but then I saw that the whole file is like this, so it would not make much sense to change it here!